### PR TITLE
Update missing Chinese translation for getting-started part

### PR DESCRIPTION
**Pull request by jianghang8421**
This PR includes:
- update missing chinese translation for getting-started part including version: latest and v2.8
- update the broken link in getting-started part including version latest and v2.8

### CODE REVIEW

**Comment by jianghang8421** [comment]
@JacieChao  Hi, could you please help me to review this PR? Thanks!

File: i18n/zh/docusaurus-plugin-content-docs/current/getting-started/installation-and-upgrade/installation-requirements/installation-requirements.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+### K3s 要求
+
+对于容器运行时，K3s 默认附带了自己的 containerd。你也可以将 K3s 配置为使用已安装的 Docker 运行时。有关在 Docker 中使用 K3s 的更多信息，请参阅 [K3s 文档](https://docs.k3s.io/advanced#using-docker-as-the-container-runtime)。
+
+Rancher 需要安装在支持的 Kubernetes 版本上。如需了解你使用的 Rancher 版本支持哪些 Kubernetes 版本，请参见 [Rancher 支持矩阵](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions)。如需指定 K3s 版本，在运行 K3s 安装脚本时，使用 `INSTALL_K3S_VERSION` 环境变量。
+
+如果你使用 **Raspbian Buster** 在 K3s 集群上安装 Rancher，请按照[这些步骤](https://rancher.com/docs/k3s/latest/en/advanced/#enabling-legacy-iptables-on-raspbian-buster)切换到旧版 iptables。
+
+如果你使用 Alpine Linux 的 K3s 集群上安装 Rancher，请按照[这些步骤](https://rancher.com/docs/k3s/latest/en/advanced/#additional-preparation-for-alpine-linux-setup) 进行其他设置。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/getting-started/installation-and-upgrade/installation-requirements/installation-requirements.md:57]: 进行前面多了一个空格

File: i18n/zh/docusaurus-plugin-content-docs/current/getting-started/installation-and-upgrade/installation-requirements/installation-requirements.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+
+更多的数量也是能够达到的，但需要更高的硬件要求。 如果您有超过 20,000 个相同类型的资源，通过 Rancher UI 加载整个列表的时间可能需要几秒钟。
+
+:::note Evolution:
+
+Rancher 的代码库不断发展，用例不断变化，Rancher 积累的经验也在不断增长。
+
+随着指导方针的准确性不断的提高并且变得更加具体，硬件要求也会发生变化。
+
+如果您发现您的 Rancher 部署不再符合列出的建议，请[联系 Rancher](https://rancher.com/contact/) 进行重新评估。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/getting-started/installation-and-upgrade/installation-requirements/installation-requirements.md:96]: 您 => 你

**Review by JacieChao — COMMENTED** [review]
please update the problem for both latest and v2.8

File: i18n/zh/docusaurus-plugin-content-docs/current/getting-started/quick-start-guides/deploy-rancher-manager/linode.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    rancher_server_url = https://rancher.xx.xx.xx.xx.sslip.io
+    workload_node_ip = yy.yy.yy.yy
+    ```
+
+8. 将以上输出中的 `rancher_server_url` 粘贴到浏览器中。在登录页面中登录（默认用户名为 `admin`，密码为在 `rancher_server_admin_password` 中设置的密码）。
+9. 使用 `quickstart/rancher/linode` 中生成的 `id_rsa` 密钥 SSH 到 Rancher Server。
+
+#### 结果
+
+两个 Kubernetes 集群已部署到你的 Hetzner 账户中，一个运行 Rancher Server，另一个为实验部署做好准备。请注意，虽然这种设置是探索 Rancher 功能的好方法，但在生产环境中，应遵循我们的高可用设置指南。用于虚拟机的 SSH 密钥是自动生成的，存储在模块目录中。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/getting-started/quick-start-guides/deploy-rancher-manager/linode.md:69]: Hetzner => Linode

**Review by JacieChao — COMMENTED** [review]
LGTM.
PR will be ready after other dependencies are ready

**Review by martyav — APPROVED** [review]

**Pull request merged**